libs/vscode-ui/components/src/lib/autocomplete/autocomplete.component.ts

```diff
@@ -114,6 +92,21 @@ export class AutocompleteComponent
       )
     ).pipe(startWith(false), debounceTime(300), shareReplay(1));
 
+    this.visibleOptions = this._options$.pipe(
+      switchMap((options) =>
+        this.control.valueChanges.pipe(
```

Comment:
* moved all setup to OnInit
* Fixed #984  with this change from `startWith('')` to `startWith(this.control.value || '')`